content/blog/2018-03-29-react-v-16-3.md

```diff
@@ -52,6 +52,8 @@ HOCs typically [pass props through](/docs/higher-order-components.html#conventio
 The new `forwardRef` API solves this problem by providing a way for us to intercept a `ref` and forward it as a normal prop:
 `embed:16-3-release-blog-post/forward-ref-example.js`
 
+[Learn more about customizing name in DevTools here.](/docs/forwarding-refs.html#customizing-name-in-devtools)
```

Comment:
Let's make this link more generic, like the ones above. (aka "Learn more about the `forwardRef` API here" and then link to /docs/forwarding-refs.html)